packages/palette/src/elements/Carousel/Carousel.story.tsx

```diff
@@ -136,3 +142,24 @@ storiesOf("Components/Carousel", module)
       />
     )
   })
+  .add("Custom rail and cells", () => {
+    return (
+      <Demo
+        widths={["100%", "100%", "100%", "100%"]}
+        Cell={React.forwardRef((props, ref) => {
```

Comment:
Curious why `forwardRef` is this needed here?

Reply:
The annotation actually tells you 😛 

![](http://static.damonzucconi.com/_capture/9gVTx7fK9rbQ.png)

Reply:
Ahh very nice 👌